realtimenet/engine.py

```diff
@@ -194,16 +198,26 @@ def run_inference_engine(
 
         # Get predictions
         prediction = inference_engine.get_nowait()
+        model_inference_end_time = time.time()
+
+        # Inference engine frame rate
+        inference_engine_fps = (1/(model_inference_end_time - model_inference_start_time))/inference_engine.step_size
+        model_inference_start_time = model_inference_end_time
```

Comment:
Getting to this point does not mean that a new prediction was produced because `inference_engine.get_nowait()` returns `None` in case the prediction wasn't ready. Here is what I think should be done:
```
if predictions is not None:
        model_inference_end_time = time.time()
        inference_engine_fps = 1 / (model_inference_end_time - model_inference_start_time)
        model_inference_start_time = model_inference_end_time
```
(no need to divide by `inference_engine.step_size` then)

Reply:
I tried and tested it, sometimes it goes above 4 ranging from 5 or 16. What should we do for that?